pallets/gear-bank/src/lib.rs

```diff
@@ -402,7 +406,15 @@ pub mod pallet {
             // `*_no_transfer` function above.
             //
             // This call does only currency trait final transfer.
-            Self::withdraw(account_id, value).unwrap_or_else(|e| unreachable!("qed above: {e:?}"));
+            Self::withdraw(account_id, value).unwrap_or_else(|e| {
+                let err_msg = format!(
+                    "Pallet::withdraw_gas: withdraw failed. \
```

Comment:
I guess you can omit here `Pallet`.
Or make it `pallet_gear_bank`, but then please adjust error logs on `pallet_gear`.

Reply:
Also would be nice to add to log info the reducible balance of `account_id`, `Bank`, also unused value and on finalize value.

Reply:
Fixed, extended log msg